### PR TITLE
Run tests in CI pipeline

### PR DESCRIPTION
**Pull request by cap10morgan**
Now that we have some tests, we should run 'em!

I made a ticket for this: FC-1001

### CODE REVIEW

**Comment by cap10morgan** [comment]
Draft until I get it working :)

**Comment by cap10morgan** [comment]
OK seems to be working now

**Review by zonotope — APPROVED** [review]
🔍

File: script/run-in-docker.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+image=fluree/${PWD##*/}
+
+echo "Running in ${image} container..."
```
Comment (jakep36) [inline script/run-in-docker.sh:7]: Where does ${image} come from when the build step is next?

**Pull request merged**